hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/container/TestContainerStateManagerIntegration.java

```diff
@@ -102,7 +100,7 @@ public void testAllocateContainer() throws IOException {
             SCMTestUtils.getReplicationFactor(conf), OzoneConsts.OZONE);
     ContainerInfo info = containerManager
         .getMatchingContainer(OzoneConsts.GB * 3, OzoneConsts.OZONE,
-            container1.getPipeline());
+            container1.getPipeline(), excludedContainerIDS);
```

Comment:
We still have older method with out needing to pass excludedContainerIDS right?

Reply:
```
//V1 code 
  ContainerInfo getMatchingContainer(long size, String owner,
      Pipeline pipeline);

//V2 code
  ContainerInfo getMatchingContainer(long size, String owner,
                                     Pipeline pipeline,
                                     Set<ContainerID> excludedContainerIDS);
```

v1 code is not used any more, so after we remove v1 code, we could used V2 method instead.

Reply:
We still have old method which is defined in interface ContainerManager.java

  default ContainerInfo getMatchingContainer(long size, String owner,
                                     Pipeline pipeline) {
    return getMatchingContainer(size, owner, pipeline, Collections.emptySet());
  }

But I think that is fine, as anyway for tests we are passing empty hashset for excludedContainerIDS